app/src/main/res/values/strings.xml

```diff
@@ -432,8 +432,13 @@
     <string name="issue_comment_hint">Comment on this issue</string>
     <string name="commit_comment_header">%1$s added a note %2$s</string>
     <string name="issue_no_description">No description provided.</string>
-    <string name="close_issue_confirm">Do you really want to close this issue?</string>
     <string name="reopen_issue_confirm">Do you really want to reopen this issue?</string>
+    <string name="choose_reason">Choose a reason</string>
```

Comment:
How about `Close issue as\u2026`?

Reply:
I agree that it suits better. I've pushed the change and updated the screenshot in the description.